hs_geographic_feature_resource/models.py

```diff
@@ -0,0 +1,390 @@
+from django.db import models
+from mezzanine.pages.page_processors import processor_for
+from hs_core.models import BaseResource, ResourceManager, resource_processor, CoreMetaData, AbstractMetaDataElement
+from mezzanine.pages.models import Page
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.contrib.contenttypes import generic
+
+# Create your models here.
+
+
+class OriginalFileInfo(AbstractMetaDataElement):
+
+    term = 'OriginalFileInfo'
+
+    fileTypeEnum=(  (None, 'Unknown'),
+                    ("SHP", "ESRI Shapefiles"),
+                    ("ZSHP", "Zipped ESRI Shapefiles"),
+                    ("KML", "KML"),
+                    ("KMZ", "KMZ"),
+                    ("GML", "GML"),
+                    ("SQLITE", "SQLite")
+                 )
+    fileType=models.TextField(max_length=128, choices=fileTypeEnum, default=None)
+    baseFilename=models.TextField(max_length=256, null=False, blank=False)
+    fileCount=models.IntegerField(null=False, blank=False, default=0)
+
+    filenameString = models.TextField(max_length=2048, null=True, blank=True)
```

Comment:
Where does the number `2048` come from and if it is arbitrary is it even needed?